lib/dot.js

```diff
@@ -0,0 +1,9 @@
+var Mustache = require('mustache')
```

Comment:
let's rename `dot.js` and any references to `dot`, no need to keep the name `dot`, something like `template.js` sounds more appropriate

Reply:
done